server/src/@common/dto/map/map-notify.dto.ts

```diff
@@ -0,0 +1,22 @@
+﻿import { ApiProperty, PickType } from '@nestjs/swagger';
+import { MapNotify } from '@prisma/client';
+import { Exclude } from 'class-transformer';
+
+export class MapNotifyDto implements MapNotify {
```

Comment:
Missing properties that are on the Prisma model

Reply:
I think your Prisma client is probably out of date, the current type is
```ts
export type MapNotify = {
  notifyOn: number
  mapID: number
  userID: number
  createdAt: Date
  updatedAt: Date
}
```